### PR TITLE
Victor example changes

### PR DESCRIPTION
**Pull request by victorwiebe**

### CODE REVIEW

File: examples/countTransferManager.ts [hunk 1]
```diff
@@ -20,16 +19,14 @@ window.addEventListener('load', async () => {
...
+  const myAddress = await polymathAPI.getAccount();
 
-  await polymathAPI.getPolyTokens({ amount: new BigNumber(1000000), address: address[0] });
-  await polymathAPI.getPolyTokens({
-    amount: new BigNumber(1000000),
-    address: await polymathAPI.securityTokenRegistry.address(),
+  // Get some poly tokens in your account and the security token
+  // Poly token faucet works on test net
+  await polymathAPI.getPolyTokens({ amount: new BigNumber(1000000) });
+  await polymathAPI.polyToken.transfer({
```
Comment (VictorVicente) [inline examples/countTransferManager.ts]: Sorry, my bad. You don't need to transfer. You only need to approve as you are doing below, then the ST calls `transferFrom`. So this entire line is not needed.

File: examples/erc20Dividend.ts [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+  };
+
+  // Instantiate the API
+  const polymathAPI = new PolymathAPI(params);
+
+  // Get some poly tokens in your account and the security token
+  const myAddress = await polymathAPI.getAccount();
+  // Token faucet on test net only
+  await polymathAPI.getPolyTokens({ amount: new BigNumber(1000000), address: myAddress });
+  await polymathAPI.polyToken.transfer({
```
Comment (VictorVicente) [inline examples/erc20Dividend.ts]: Same as above. You don't need to transfer. You only need to approve as you are doing below, then the ST calls `transferFrom`. So this entire line is not needed.

File: examples/erc20Dividend.ts [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+
+  const erc20DividendCheckpoint = await polymathAPI.moduleFactory.getModuleInstance({
+    name: ModuleName.erc20DividendCheckpoint,
+    address: erc20DividendAddress,
+  });
+  const erc20Dividend = (await tickerSecurityTokenInstance.getModulesByName({
+    moduleName: ModuleName.erc20DividendCheckpoint,
+  }))[0];
+
+  await polymathAPI.polyToken.transfer({ to: erc20Dividend, value: new BigNumber(5) });
```
Comment (VictorVicente) [inline examples/erc20Dividend.ts]: You don't need to transfer, with the approve from next line is ok.

File: examples/etherDividend.ts [hunk 4]
```diff
@@ -22,14 +23,11 @@ window.addEventListener('load', async () => {
...
-    gasPrice: params.defaultGasPrice,
-  });
-  const address = await web3Wrapper.getAvailableAddressesAsync();
-  await polymathAPI.getPolyTokens({ amount: new BigNumber(1000000), address: address[0] });
-  await polymathAPI.getPolyTokens({
-    amount: new BigNumber(1000000),
-    address: await polymathAPI.securityTokenRegistry.address(),
+  const myAddress = await polymathAPI.getAccount();
+  await polymathAPI.getPolyTokens({ amount: new BigNumber(1000000), address: myAddress });
+  await polymathAPI.polyToken.transfer({
```
Comment (VictorVicente) [inline examples/etherDividend.ts]: Same as above. You don't need to transfer. You only need to approve as you are doing below, then the ST calls `transferFrom`. So this entire line is not needed.

File: examples/generalPermissionManager.ts [hunk 5]
```diff
@@ -21,16 +21,13 @@ window.addEventListener('load', async () => {
...
-  const address = await web3Wrapper.getAvailableAddressesAsync();
 
-  await polymathAPI.getPolyTokens({ amount: new BigNumber(1000000), address: address[0] });
-  await polymathAPI.getPolyTokens({
-    amount: new BigNumber(1000000),
-    address: await polymathAPI.securityTokenRegistry.address(),
+  // Get some poly tokens in your account and the security token
+  const myAddress = await polymathAPI.getAccount();
+  await polymathAPI.getPolyTokens({ amount: new BigNumber(1000000), address: myAddress });
+  await polymathAPI.polyToken.transfer({
```
Comment (VictorVicente) [inline examples/generalPermissionManager.ts]: Same as above. You don't need to transfer. You only need to approve as you are doing below, then the ST calls `transferFrom`. So this entire line is not needed.

File: examples/generalTransferManager.ts [hunk 6]
```diff
@@ -20,16 +20,13 @@ window.addEventListener('load', async () => {
...
-  const address = await web3Wrapper.getAvailableAddressesAsync();
 
-  await polymathAPI.getPolyTokens({ amount: new BigNumber(1000000), address: address[0] });
-  await polymathAPI.getPolyTokens({
-    amount: new BigNumber(1000000),
-    address: await polymathAPI.securityTokenRegistry.address(),
+  // Get some poly tokens in your account and the security token
+  const myAddress = await polymathAPI.getAccount();
+  await polymathAPI.getPolyTokens({ amount: new BigNumber(1000000), address: myAddress });
+  await polymathAPI.polyToken.transfer({
```
Comment (VictorVicente) [inline examples/generalTransferManager.ts]: Same as above. You don't need to transfer. You only need to approve as you are doing below, then the ST calls `transferFrom`. So this entire line is not needed.

File: examples/usdTieredSTO.ts [hunk 7]
```diff
@@ -24,14 +22,11 @@ window.addEventListener('load', async () => {
...
-    gasPrice: params.defaultGasPrice,
-  });
-  const address = await web3Wrapper.getAvailableAddressesAsync();
-  await polymathAPI.getPolyTokens({ amount: new BigNumber(1000000), address: address[0] });
-  await polymathAPI.getPolyTokens({
-    amount: new BigNumber(1000000),
-    address: await polymathAPI.securityTokenRegistry.address(),
+  const myAddress = await polymathAPI.getAccount();
+  await polymathAPI.getPolyTokens({ amount: new BigNumber(1000000), address: myAddress });
+  await polymathAPI.polyToken.transfer({
```
Comment (VictorVicente) [inline examples/usdTieredSTO.ts]: Same as above. You don't need to transfer. You only need to approve as you are doing below, then the ST calls `transferFrom`. So this entire line is not needed.

File: examples/volumeRestrictionTransferManager.ts [hunk 8]
```diff
@@ -23,15 +22,11 @@ window.addEventListener('load', async () => {
...
-  });
-  const address = await web3Wrapper.getAvailableAddressesAsync();
-
-  await polymathAPI.getPolyTokens({ amount: new BigNumber(1000000), address: address[0] });
-  await polymathAPI.getPolyTokens({
-    amount: new BigNumber(1000000),
-    address: await polymathAPI.securityTokenRegistry.address(),
+  const myAddress = await polymathAPI.getAccount();
+  await polymathAPI.getPolyTokens({ amount: new BigNumber(1000000), address: myAddress });
+  await polymathAPI.polyToken.transfer({
```
Comment (VictorVicente) [inline examples/volumeRestrictionTransferManager.ts]: Same as above. You don't need to transfer. You only need to approve as you are doing below, then the ST calls `transferFrom`. So this entire line is not needed.

File: examples/countTransferManager.ts (hunk 1 above)
Comment (victorwiebe) [reply examples/countTransferManager.ts]: Okay I tested it and saw that, I will go ahead and just remove that transfer from my files

**Pull request merged**